app/presenters/prior_authority/tasks/primary_quote.rb

```diff
@@ -0,0 +1,16 @@
+module PriorAuthority
+  module Tasks
+    class PrimaryQuote < ::Tasks::Generic
+      PREVIOUS_TASK = CaseAndHearingDetail
```

Comment:
I know that CaseAndHearingDetail _is_ the previous task, but what this config actually does is say what has to be completed before PrimaryQuote can be started. And I don't think we actually need them to complete that to start this. If you look in PriorAuthority::Tasks::ClientDetail you'll see we've worked around this there by lying about the previous task but adding an explanatory comment. I wonder if it would be worth doing that here.

Reply:
yeah I think this is necessary context - before I do this I'm going to look into whether maybe there's actually an explicit can_start? method that I could put together because thinking about it, when the flow is on "non prison law", there are other forms that need completing before we can get to primary quote I think  like youth court/psychiatric liason (I'll double check in ticket and with Mary)

Reply:
yeah after talking with Mary, the "last step" of this section depends on whether Prison Law is Yes or No so I'm going to make an explicit can_start? override instead of relying on PREVIOUS_TAST